test/server/script-processor-test.js

```diff
@@ -943,5 +943,12 @@ describe('Script processor', () => {
                 expected: 'async function foo() {  return __get$(obj, "src"); }'
             });
         });
+
+        it('Should process the `super` expression (GH-1390)', () => {
+            testProcessing({
```

Comment:
Do we cover the `super.prop` expression?

Reply:
Yes

Reply:
Ok